Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostNic/VMHostVdsNic.ps1

```diff
@@ -0,0 +1,181 @@
+<#
+Copyright (c) 2018 VMware, Inc.  All rights reserved
+
+The BSD-2 license (the "License") set forth below applies to all parts of the Desired State Configuration Resources for VMware project.  You may not use this file except in compliance with the License.
+
+BSD-2 License
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#>
+
+[DscResource()]
+class VMHostVdsNic : VMHostNicBaseDSC {
+    <#
+    .DESCRIPTION
+
+    Specifies the name of the Distributed Switch to which the VMKernel Network Adapter should be connected.
+    #>
+    [DscProperty(Key)]
+    [string] $VdsName
+
+    <#
+    .DESCRIPTION
+
+    Specifies the port of the specified Distributed Switch to which the VMKernel Network Adapter should be connected.
+    #>
+    [DscProperty()]
+    [string] $PortId
```

Comment:
As we discussed offline, PortId doesn't make sense to be configurable because VDS "generates" them by unknown logic.
Niv Ensure=Absent doesn't work on Nic but on portgroup, so consider removing Absent option or Ensure property.